messages/bg.json

```diff
@@ -558,6 +558,8 @@
   "admin/store.advancedSettings.enableLazyFooter.description": "Бавно рендиране на долен колонтитул на страница и бавно зареждане на активите на страницата, което подобрява представянето на първото рендиране на страницата.",
   "admin/store.advancedSettings.enableLazyFold.title": "Активиране на бавно зареждане на активите под ръба на първия екран",
   "admin/store.advancedSettings.enableLazyFold.description": "Активира бавно зареждане на JS файлове на компонентите под блок „__fold__“.",
+  "admin/store.advancedSettings.fetchSponsoredProductsOnSearch.title": "[БЕТА] Извличане на спонсорирани продукти в резултатите от търсенето.",
```

Comment:
were these messages reviewed by the product localization team?

Reply:
@thalytafabrine No, they weren't. I just used Google Translate 😢 

What do you guys usually do? It's not important to have these many translations, but I didn't want to add warnings for lack of translations.